pkg/releaser/git.go

```diff
@@ -123,23 +129,39 @@ func getPushRefSpec(branchName string) string {
 	return fmt.Sprintf("refs/heads/%s:refs/heads/%s", branchName, branchName)
 }
 
+func getGitHubClient(token string) *github.Client {
+	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
+	tc := oauth2.NewClient(context.TODO(), ts)
+	return github.NewClient(tc)
+}
+
+func (r *Releaser) getRepo(client *github.Client, owner string, repoName string) (*github.Repository, error) {
+	repo, _, err := client.Repositories.Get(context.TODO(), owner, repoName)
+    if err != nil {
+        return nil, err
+    }
+	return repo, nil
+}
+
 // SubmitPR submits the PR
 func (r *Releaser) submitPR(request *source.ReleaseRequest) (string, error) {
-	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: r.Token})
-	tc := oauth2.NewClient(context.TODO(), ts)
-	client := github.NewClient(tc)
+	client := getGitHubClient(r.Token)
```

Comment:
maybe have a client on Releaser object and just reuse that instead of creating here and in clonerepo function. 

also it seems like could needs indentation.